mixer/tools/codegen/cmd/mixgenbootstrap/main.go

```diff
@@ -61,6 +61,9 @@ func withArgs(args []string, errorf func(format string, a ...interface{})) {
 			importMapping := make(map[string]string)
 			for _, maps := range mappings {
 				m := strings.Split(maps, ":")
+				if len(m) != 2 {
+					errorf("Invalid map %s, map should only contain one colon", maps)
```

Comment:
@zhuguihua thanks for fixing this. 
Nit comment: Consider elaborating the error message.
errorf("Invalid argument '%s'. Argument should contain one colon.
							Example: -m google/protobuf/descriptor.proto:github.com/golang/protobuf/protoc-gen-go/descriptor")